module/src/main/java/com/annimon/tgbotsmodule/commands/context/MessageContext.java

```diff
@@ -9,15 +9,11 @@
 import org.jetbrains.annotations.NotNull;
 import org.telegram.telegrambots.meta.api.objects.Message;
 import org.telegram.telegrambots.meta.api.objects.Update;
-import org.telegram.telegrambots.meta.api.objects.User;
 
 public class MessageContext extends Context {
 
-    private final Long chatId;
```

Comment:
chatId should remain a field in this class as it doesn't change across calls and it's often used